tests/add.bats

```diff
@@ -28,11 +28,13 @@ load helpers
   # Copy a file to a specific subdirectory
   run_buildah add $cid ${TEST_SCRATCH_DIR}/randomfile /subdir
   # Copy two files to a specific subdirectory
-  run_buildah add $cid ${TEST_SCRATCH_DIR}/randomfile ${TEST_SCRATCH_DIR}/other-randomfile /other-subdir
+  run_buildah add $cid ${TEST_SCRATCH_DIR}/randomfile ${TEST_SCRATCH_DIR}/other-randomfile /other-subdir/
   # Copy two files to a specific location, which succeeds because we can create it as a directory.
-  run_buildah add $cid ${TEST_SCRATCH_DIR}/randomfile ${TEST_SCRATCH_DIR}/other-randomfile /notthereyet-subdir
+  run_buildah add $cid ${TEST_SCRATCH_DIR}/randomfile ${TEST_SCRATCH_DIR}/other-randomfile /notthereyet-subdir/
+  # Add with a wildcard and destination dir does not exists
+  run_buildah add $cid ${TEST_SCRATCH_DIR}/* /notthereyet2-subdir/
```

Comment:
Without quoting, the globbing here is done by the shell, not by buildah, so buildah doesn't see the globbing pattern.